### DISCUSSION TITLE
Daily Standup (Friday, June 10th)

### DISCUSSION THREAD

**Discussion by CristianCazares** [CarlosCRG19/ijalti-peers#123] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much, only code review

**What are your priorities for the next 24 hours?**:
 - Finish #121 

**Blockers**:
 - I am very sleepy and tired :c

**Shoutouts**:
 - @CarlosCRG19 because I am going to miss him

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on #103 

**What are your priorities for the next 24 hours?**:
 - Finish #103 

**Blockers**:
 - Some MUI problems

**Shoutouts**:
 - @CristianCazares

**Reply by BrandonMagana** [reply]
**What did you achieve in the last 24 hours?**:
 - worked on pagination feature 

**What are your priorities for the next 24 hours?**:
 - don't now yet

**Blockers**:
 - None

**Shoutouts**:
 - @everyone

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Started #108 . Working on the job offer cards at the moment.

**What are your priorities for the next 24 hours?**:
 - Finished all the frontend related stuff of the feed

**Blockers**:
 - None. So far so good

**Shoutouts**:
 - @everyone for the hard work
